src/olympia/activity/serializers.py

```diff
@@ -29,7 +29,8 @@ def get_comments(self, obj):
 
     def get_action_label(self, obj):
         log = obj.log()
-        return _(u'Review note') if not hasattr(log, 'short') else log.short
+        default = ugettext(u'Review note')
```

Comment:
I doubt it was intendedly a `ugettext_lazy` before but by accident. All other instances of `get_action_label`s result are being used as a regular string so laziness didn't do anything here.